docs: add `waitForResponse` flag documentation (#57)
athombv/node-zigbee-clusters: README.md

          .then(async node => {
            // Create ZCLNode instance
            const zclNode = new ZCLNode(node);

            // Send toggle command to onOff cluster on endpoint 1
            await zclNode.endpoints[1].clusters[CLUSTER.ON_OFF.NAME].toggle();

            // Send moveToLevel command to levelControl cluster on endpoint 1 and don't wait for
            // the default response confirmation.
            await zclNode.endpoints[1].clusters[CLUSTER.LEVEL_CONTROL.NAME].moveToLevel({
              level: 100,
              transitionTime: 2000
            }, {
              // This is an optional flag that disables waiting for a default response from the
              // receiving node as a confirmation that the command is received and executed.

>               // Usually you don't want to add this flag, only if the device does not follow the
>               // Zigbee specification and refuses to send a default response this flag is
>               // recommended.

Reviewer: ```suggestion
              // You should only use this flag if the device does not follow the
              // Zigbee specification and refuses to send a default response.
```

Author: Better, thanks!
